aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,142 @@
+  docs="https://v2.grommet.io/starrating?theme=hpe"
+  figma="https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5554%3A77765&t=TNCIYkwaIfFrdWxf-1"
+>
+  <SimpleFeedbackExample />
+</Example>
+
+## When to use
+When and where to collect feedback from users should be a collaborative effort working with the <a href={`mailto:${contact.email}?subject=Collecting%20user%20feedback%20in%20a%20product%20experience`}>unified customer experience team</a>.
+This team will work with you to construct a questionnaire aligned with your objectives and 
+[provide implementation details](#feedback-setup-and-submission) such as API credentials and endpoints for submitting 
+feedback responses.
+
+In-app feedback collection should be considered where it aids the understanding of users' 
+perceptions around the ease-of-use and/or satisfaction with particular content, features, 
+or workflows.
+
+
+Feedback may be [actively solicited](#solicited-feedback) or [passively collected](#capturing-unsolicited-feedback).
+
+### When not to use
+Don’t collect feedback in the following use cases:
+- If it disrupts a user flow or is intrusive in any way.
+- There is no actionable plan for what to do with the data.
+- In locations where users do not have enough context to form a valid opinion. 
+- There are too many instances soliciting feedback which can [dilute feedback effectiveness](#sampling-frequency).
+
+### Sampling frequency
+Avoid over sampling and/or diluting effectiveness by limiting who is asked for feedback and 
+how often. Consider implementing "rules of engagement" specifying under what conditions an 
+individual user should be solicited for feedback. Example engagement rules might be:
+- Has not submitted response to this questionnaire in the last 90 days.
+- Has not been asked to complete this questionnaire in the last 30 days.
+- Has not been asked to complete any questionnaire in the last 7 days.
+
+## Anatomy
+<Example plain>
+  <FeedbackAnatomy />
+</Example>
+
+| Label  | Region       | Purpose                                                              |                  Required                   | Notes                                                                                                      |
+| ------ | ------------ | -------------------------------------------------------------------- | :-----------------------------------------: | ---------------------------------------------------------------------------------------------------------- |
+| **1**  | **Title**    | Title prompting user for their feedback.                             | <Checkmark a11yTitle="true" size="small" /> | <span>Use the [semantically correct heading level](/foundation/typography?q=semantically#semantic-usage-of-heading-levels) for the title. "We'd love your feedback" is a nice, appreciative default title.</span>       |
```

Comment:
I do love the idea of replacing text with an icon within the table here. For consistency with the rest of our icons, should we have those be text-strong as well as our default stroke weight? These look a littler thinner.

Reply:
so I saw were using `checkmark` from grommet-icons for the tables and the thicker one is `FormCheckmark` so going to leave it as `checkmark` for now so its consistent across the tables but something for us to consider which one to use